### PR TITLE
Update examples for "SameSite cookies explained blog"

### PR DESCRIPTION
**Pull request by rowan-m**
- Examples have been moved to their own repo at https://github.com/GoogleChromeLabs/samesite-examples
- Reference added for Safari bug along with explanation
- Added `alt` text for images
 -Switched to `.example` for mock domains
- Cleaned up formatting

### CODE REVIEW

**Comment by rowan-m** [comment]
Staged at: https://20190613t130249-dot-web-dev-staging.appspot.com/samesite-cookies-explained/

**Comment by rowan-m** [comment]
Open question: should I update the `date`?

File: src/site/content/en/blog/samesite-cookies-explained/index.md [hunk 1]
```diff
@@ -268,68 +268,77 @@ on your site, you should also check with the provider that they are updating
 their services. You may need to update your dependencies or snippets to ensure
 that your site picks up the new behavior. {% endAside %}
 
-These changes are all backwards-compatible with other browsers. By making these
-changes to your cookies, you are making their intended use explicit rather than
-relying on the default behavior of the browser. Likewise, any clients that do
-not recognize `SameSite=None` as of yet should ignore it and carry on as if the
-attribute was not set.
+These changes are backwards-compatible with browsers that have correctly
```
Comment (Meggin) [inline src/site/content/en/blog/samesite-cookies-explained/index.md]: Language a little funky... what about 'that have correctly implemented earlier versions of the `SameSite` attribute or that do not'

File: src/site/content/en/blog/samesite-cookies-explained/index.md [hunk 2]
```diff
@@ -268,68 +268,77 @@ on your site, you should also check with the provider that they are updating
...
+do not recognize `SameSite=None` as of yet should ignore it and carry on as if
+the attribute was not set.
 
 Additionally in Chrome, if you also enable the
 `cookies-without-same-site-must-be-secure` flag then you must also specify
 `SameSite=None` cookies as `Secure` or they will be rejected. Note, this flag
 won't have any effect unless you also have `same-site-by-default-cookies`
 enabled.
 
+{% Aside 'warning' %} Warning: At the time of writing, the network library on
```
Comment (Meggin) [inline src/site/content/en/blog/samesite-cookies-explained/index.md]: Remove the "Warning:" as it is part of aside formatting (duplicate now). Also, replace 'You can track' with just 'Track' to reduce size of aside.

Comment (rowan-m) [reply src/site/content/en/blog/samesite-cookies-explained/index.md]: Oh, yes! That was a copy/paste error - thank you!

File: src/site/content/en/blog/samesite-cookies-explained/index.md (hunk 1 above)
Comment (rowan-m) [reply src/site/content/en/blog/samesite-cookies-explained/index.md]: Hmm... looks like I lost part of the sentence somewhere! :O

**Comment by rowan-m** [comment]
> There's a good bit deleted from this post, which I'm guessing is as intended. I can't verify accuracy though. Approving much the same, with the notion that it's as intended.

I deleted a bit about bloating headers with cookies resulting in more request packets, but it's really not a major concern and we're also adding other security headers anyway so it was distracting from the main message. Not sure if that's the bit you were referring to though.

**Comment by rowan-m** [comment]
Will make these corrections in the morning.

**Comment by rowan-m** [comment]
All corrections and updates done, ready for a final pass! @Meggin @robdodson

**Comment by Meggin** [comment]
Already approved. Merge :).

**Pull request merged**